src/pages/resultsView/cancerSummary/CancerSummaryContainer.tsx

```diff
@@ -101,42 +134,15 @@ export default class CancerSummaryContainer extends React.Component<{ store: Res
     }
 
     public render() {
```

Comment:
log of the rending stuff proved very painful to unit test so i focused on covering it in e2e's